tests/baselines/reference/contextualSignatureInArrayElementPrefersArrayUnionMemberLibEs5.errors.txt

```diff
@@ -0,0 +1,18 @@
+tests/cases/compiler/contextualSignatureInArrayElementPrefersArrayUnionMemberLibEs5.ts(8,4): error TS7006: Parameter 'arg' implicitly has an 'any' type.
+
+
+==== tests/cases/compiler/contextualSignatureInArrayElementPrefersArrayUnionMemberLibEs5.ts (1 errors) ====
+    // repro from #52588

```

Comment:
I'm also not sure what's your policy for those comments - but a future reader might find this reference confusing (as the issue is about making this work but this new test case will test that it **doesn't** work). I would personally either remove this line (from the test case file) or adjust it to point to Ander's comment: `53280#discussion_r1138744885`

Reply:
I'm swapping them out to be real links, so this will be updated in a moment.

I don't know if there's any policy; my personal preference is to leave real actual links that I can click on, but most don't do that. Even more, I wish we just named tests like `issue53280.ts` rather than having to come up with some readable name that nobody's ever going to change again even if the behavior changes.